flixel/util/FlxStringUtil.hx

```diff
@@ -262,14 +262,8 @@ class FlxStringUtil
 	 */
 	public static function getDomain(url:String):String
```

Comment:
Eh, I'm inclined to leave that alone for now. Changing it would be a breaking change - we can't guruantee that only `FlxBasePreloader` relies on it, other people might use it as well.